custom_components/braviatv_psk/manifest.json

```diff
@@ -2,7 +2,8 @@
   "domain": "braviatv_psk",
   "name": "Bravia TV PSK",
   "documentation": "https://github.com/custom-components/media_player.braviatv_psk",
-  "requirements": ["pySonyBraviaPSK==0.2.0"],
+  "requirements": ["git+https://github.com/JackED42/braviarc.git@master#pySonyBraviaPSK==2.0"],
```

Comment:
I could help with testing this but I wonder if you have created a PR against pySonyBraviaPSK with your changes also?

Reply:
Yeh I raised pr, didn't hear anything from the developer...

Reply:
Did you raise it in the right place? I can't see anything in https://github.com/gerard33/sony_bravia_psk/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc